patches/llvm-project/0001-libc-tests-with-picolibc-xfail-two-remaining-tests.patch

```diff
@@ -1,7 +1,7 @@
-From cb6cfe0e5fc29d3867f523900f0d46a4ef5dff1b Mon Sep 17 00:00:00 2001
+From 9fb80381b839fdbe15ddfa732bed5c25d6e4a5a2 Mon Sep 17 00:00:00 2001
 From: =?UTF-8?q?Dominik=20W=C3=B3jt?= <dominik.wojt@arm.com>
 Date: Mon, 16 Oct 2023 11:35:48 +0200
-Subject: [libc++] tests with picolibc: xfail two remaining tests
+Subject: [PATCH 1/9] [libc++] tests with picolibc: xfail two remaining tests
```

Comment:
You should remaster the patch series using `format-patch -k`, to avoid getting these `[PATCH 1/9]` prefixes. Goes with the `git am -k` we advise for reapplying them.

(If you didn't do either, then `git am` without `-k` would also discard square-bracketed prefixes you actually wanted to keep, which there are lots of in LLVM.)